src/test/java/org/jenkinsci/plugins/workflow/support/steps/ExecutorStepTest.java

```diff
@@ -422,4 +424,36 @@ private void startJnlpProc() throws Exception {
         });
     }
 
+    @Issue("JENKINS-26132")
+    @Test public void enclosingLabel() {
```

Comment:
Looks pretty solid to me;  in the future we may wish to add coverage of nested  & parallel labels just to ensure that works correctly, but for stages... not an issue.